.github/main.workflow

```diff
@@ -13,3 +13,19 @@ action "Milestone It" {
   needs = ["Filter merged"]
   secrets = ["GITHUB_TOKEN"]
 }
+
+workflow "Assign fixed issues when pull request opened" {
+  on = "pull_request"
+  resolves = ["Assign Fixed Issues"]
+}
+
+action "Filter opened" {
+  uses = "actions/bin/filter@0dbb077f64d0ec1068a644d25c71b1db66148a24"
+  args = "action opened"
```

Comment:
Right now I'm only handling the case where a pull request is opened. If one edited an existing pull request to include a 'Fixes #XXXX', the action would not pick it up. I'm thinking that this is fine for now and that we can build our way up once we're more confident in the action.

Reply:
I think to avoid annoying Github filters links in the PR @aduth  moved these filters to the entry point script instead. Should we do the same?

Reply:
My changes in #16702 weren't related to the spamming of the checks box. I did replace the filtering, partly on a hunch that it might explain inadvertent cancellations we've been seeing when using GitHub's canonical `actions/bin` repository (the cancellations occur in this container).

Reply:
Do you think I should have any filtering in the entry point script here?

Reply:
> Do you think I should have any filtering in the entry point script here?

It's hard to say. In a perfect world, the way you've implemented it would be correct. The problem we were seeing which led to #16702 was that this step was frequently being cancelled unexpected. I have an open ticket with GitHub support which they've repeatedly deferred, so the changes there were meant as an experiment on our end to determine whether it might provide any more stability to our GitHub Actions workflows.

So far: The cancellations have stopped. But now we're seeing another new issue that they've become perpetually queued ¯\\_(ツ)\_/¯ [[1]](https://github.com/WordPress/gutenberg/pull/16783/checks) [[2]](https://github.com/WordPress/gutenberg/pull/16259/checks)

For the purposes of your pull request, I'd say to keep it as you've implemented, but expect that it unfortunately may not work very reliably.

Reply:
To the original question: I think your proposal as it stands now is reasonable, and if we want additional coverage later, it can come as a future enhancement.